docs/Contributing.md

```diff
@@ -1,4 +1,4 @@
-# Contributing
+# Contributing to `ros2_rust`
 Contributions to `ros2_rust` are very welcome!
 
 Work is coordinated in the [Matrix chat](https://matrix.to/#/+rosorg-rust:matrix.org). It's a good idea to check in with the other contributors there before starting development on a new feature.
```

Comment:
While Matrix is nice, I personally prefer if the focus is on GitHub. It's hard to keep track of conversations on Matrix, and there's no trail of who is working on what. So I'd advise contributors to first check GitHub, file an issue for what they want to work on, and then perhaps talk about it on Matrix, but again, technical discussions on a chat are hard to follow.

Reply:
Ok, I rephrased this.